crates/viewer/re_view_spatial/src/eye.rs

```diff
@@ -411,6 +375,19 @@ impl ViewEye {
             }
         });
 
+        let kind = eye_property.component_or_fallback::<Eye3DKind>(
+            view_ctx,
+            self,
+            &Eye3D::descriptor_kind(),
+        );
+        match kind {
+            Ok(Eye3DKind::FirstPerson) => self.set_kind(Eye3DKind::FirstPerson),
+            Ok(Eye3DKind::Orbital) => self.set_kind(Eye3DKind::Orbital),
+            Err(err) => {
+                re_log::error!("error while getting eye 3d kind: {}", err);
```

Comment:
```suggestion
                re_log::error_once!("error while getting eye 3d kind: {}", err);
```

don't think we'll ever hit it, but might get spammy quickly if we do